deploy/eck-stack/examples/kibana/http-configuration.yaml

```diff
@@ -0,0 +1,42 @@
+---
+eck-kibana:
+  # Name of the Kibana instance.
+  #
+  name: "quickstart"
+
+  # Version of Kibana.
+  #
+  version: 8.2.3
+
+  # Labels that will be applied to Kibana.
+  #
+  labels: {}
+    # key: value
+
+  # Annotations that will be applied to Kibana.
+  #
+  annotations: {}
+    # key: value
+
+  spec:
+    # Count of Kibana instances to create.
+    #
+    count: 1
+
+    # Reference to ECK-managed Elasticsearch instanced, ideally from {{ .Values.elasticsearch.name }}
+    #
+    elasticsearchRef:
+      name: quickstart
+      namespace: default
+    http:
+      service:
+        spec:
+          # Type of service to deploy for Kibana.
+          # This deploys a load balancer in a cloud service provider, where supported.
+          # 
+          type: LoadBalancer
+      tls:
+        selfSignedCertificate:
+          subjectAltNames:
+          - ip: 1.2.3.4
+          - dns: kibana.example.com
```

Comment:
Should we leave the `subjectAltNames` configuration  commented given that is not a working configuration?